docs/administration.md

```diff
@@ -227,19 +227,26 @@ is not a TTY" errors. For example:
 `docker-compose exec -T webserver document_exporter ../export`
 
 ```
-document_exporter target [-c] [-f] [-d]
+document_exporter target [-sm] [-c] [-f] [-p] [-d] [-na] [-nt]
 
 optional arguments:
+-sm, --split-manifest
```

Comment:
I don't think there was a plan with these before, but I think it would be nice to see the options sorted alphabetically, by the short name

Reply:
fixed by 202d8a70884d76d0abf5ea37e65dba1b20b78f18